static/app/views/performance/transactionSummary/transactionThresholdModal.tsx

```diff
@@ -294,7 +294,6 @@ const Instruction = styled('div')`
 export const modalCss = css`
   width: 100%;
   max-width: 650px;
-  margin: 70px auto;
```

Comment:
Leaving this alone caused a visual snapshot regression, i pulled it out because it's not really doing much. If there are regressions anyway then i'd be ok with that because the position of this modal is more consistent.